examples/pulse_control/6_Use_jobs_with_pulse_sequence_sweeps.ipynb

```diff
@@ -0,0 +1,409 @@
+{
```

Comment:
<p>1) you <strong>can</strong> always recreate it</p><p> </p><p>2) Hybrid Job ARN</p><p> </p><p>3) <strong>Using</strong> the region corresponding to your device</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/294/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>